### ISSUE TITLE
Setting Where The Gcode Cuts First.

### ISSUE THREAD

**Issue by Akfreak** [tbfleming/jscut#43]
I am using JSCut to setup OX CNC gantry plates. The parts are beautiful, and everything is almost perfect. However the fist cut on all the parts is the outside boarder of the part. How can I set it so it does all the holes and pocket cuts inside the part first, the finishes with the outside last?    

If the outside is cut first, the clamping is a mute point at the part is now loose for all the rest of the operations. Please tell me how to fix this problem.   PS, I love JS cut. Great job!

**Comment by tbfleming** [comment]
Separate the holes from the outside shape (Inkscape's break apart command). In jscut, select the holes first, create a pocket operation, then select the outside and create an outside operation.

**Comment by tbfleming** [comment]
![screen shot 2015-06-11 at 9 39 48 am](https://cloud.githubusercontent.com/assets/418464/8108429/4e021940-101e-11e5-8197-7ffcd54aafba.png)

**Comment by Akfreak** [comment]
I use illustrator to make all my parts So I guess Ill make the plate one part, then all the holes separate parts. 

What I did was use the Pathfinder tool in Illustrator to exclude the holes from the plate and it changes into as a single part with holes in it.   

I think I know what to do now.. Thanks for your time.  BTW, I started a JScut G+ community today to talk about JScut projects ans such.  I linked all the known resources of JScut. Thanks for making this awesome and easy to use CAM tool.  Link the G+ community https://plus.google.com/communities/111432161531722713359
